deploy/README.md

```diff
@@ -81,7 +113,41 @@ For eg.
 az deployment sub create -l <region> -n aoi -f main.bicep -p location=<region> environmentCode=aoi environment=synapse-aoi
 ```
```

Comment:
I believe we had a discussion on having this or getting away from this. I remember the decision was to retain this. May be we can revisit in future PR and remove it if needed.

Reply:
The discussion we had was to retain the install script, this comment is on whether we need to have the `az deployment ...` command explicitly written on the readme because the install script already covers it.

Reply:
Yes, this was a different conversation from back in when the repo was private.

Reply:
Lets take this about in a separate discussion and PR. Not related to this PR